config/routes.rb

```diff
@@ -545,7 +545,8 @@
 
   match 'host_statuses' => 'react#index', :via => :get
   constraints(id: /[^\/]+/) do
-    match 'experimental/hosts/:id' => 'react#index', :via => :get, :as => :host_details_page
+    get 'legacy/hosts/:id' => 'hosts#show'
+    match 'hosts/:id' => 'react#index', :via => :get, :as => :host_details_page
```

Comment:
The tests are failing on this, could we flip to something like:
```suggestion
    get 'hosts/:id' => 'hosts#show'
    match 'fancy/hosts/:id' => 'react#index', :via => :get, :as => :host_details_page
```

Reply:
how about `/old/hosts/:id`?

Reply:
I'm saying that change of the URL of the old page make the tests to fail as they expect the url to be `/hosts/:id` that can be fixed in tests tho if you want.
But going with the new UI being on `/hosts/:id` makes it feel like its already default UI :)